### DISCUSSION TITLE
Standalone without thermostat

### DISCUSSION THREAD

**Discussion by benalla** [rvdbreemen/OTGW-firmware#87] [Q&A] [answered]
Hi all,

I have a Vaillant hrExclusief VHR 35C, using it with a Toon thermostat.
The last months I started using HomeAssistant.

Now I want to remove my thermostat, and use something else which can be placed out of sight.
For controlling (on/off ) HA will be used.
I have temp sensors which can be used to maintain the right temperature.

Is OTGW something what I van use? With the firmware from Robert?
Can someone help me getting started?

Thanks in advance!

**Comment by rvdbreemen** [comment, marked as answer]
Hi @benalla 
There is a community member on Discord who started a project called Smart Autotune Thermostat. It’s based on Homes Assistant automation and the OTGW to control the boiler. 

So it sounds a lot like what you are looking for. Talk to him about it. 

Just go to the project channel: https://discord.gg/TjKBH248eK

cu on discord
Robert

**Reply by benalla** [reply]
Thanks Robert.
Will do.
